docs/configuration/data-validation.md

```diff
@@ -96,6 +96,7 @@ The following colors are supported:
 
 This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. An example is provided below:
 
+**Example prevents the deletion of a site entry:**
```

Comment:
Please change this to regular text, extending the paragraph above. Text emphasis should be used sparingly.